src/info.cc

```diff
@@ -0,0 +1,29 @@
+#include "info.h"
+
+Info::Info() {
+  set_hexpand(false);
+  set_halign(Gtk::Align::ALIGN_END);
+  
+  auto content_area=dynamic_cast<Gtk::Container*>(get_content_area());
+  label.set_max_width_chars(40);
+  label.set_line_wrap(true);
+  content_area->add(label);
+    
+  auto provider = Gtk::CssProvider::create();
+  provider->load_from_data("* {border-radius: 5px;}");
+  get_style_context()->add_provider(provider, GTK_STYLE_PROVIDER_PRIORITY_APPLICATION);
+}
+
+void Info::print(const std::string &text) {
+  if(!enabled)
```

Comment:
Not sure if timeout is the way to go then. Enable/disable maybe, but still, it gives information that maybe for instance plugins want to give, so disabling does not make sense in that respect. Let's discuss this later, and prioritise other things now:)